build-and-test.ps1

```diff
@@ -6,7 +6,7 @@ $dockerRepo="microsoft/dotnet-nightly"
 pushd $PSScriptRoot
 
 Get-ChildItem -Recurse -Filter Dockerfile | where {$_.DirectoryName.TrimStart($PSScriptRoot) -like "*\nanoserver*"} | sort DirectoryName | foreach {
-    $tag = "$($dockerRepo):" + $_.DirectoryName.Replace($PSScriptRoot, '').TrimStart('\').Replace('\', '-')
+    $tag = "$($dockerRepo):" + $_.DirectoryName.Replace($PSScriptRoot, '').Replace("\nanoserver", '').TrimStart('\').Replace('\', '-') + "-nanoserver"
```

Comment:
I go back and forth on this topic.  I don't see a real benefit either way.  This is the pattern we used before so I kept it.  The nice thing about this pattern is that you can go to the repo and easily browse by platform.  Not making any changes at this time.